### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by szepeviktor**
Found few misspellings.

(`CHANGELOG.md` is excluded)

## Summary by CodeRabbit

- **Bug Fixes**
	- Corrected spelling errors across various files for improved clarity and consistency.
- **Tests**
	- Minor modifications to test descriptions for enhanced clarity.
- **Style**
	- Updated comments in SCSS files for consistency and typo correction.
- **Documentation**
	- Updated comments in TypeScript definition file for language consistency and typo correction.

### CODE REVIEW

File: sweetalert2.d.ts [hunk 1]
```diff
@@ -1129,7 +1129,7 @@ declare module 'sweetalert2' {
      * ```
      * Swal.fire({
      *   input: 'email',
-     *   validationMessage: 'Adresse e-mail invalide'
+     *   validationMessage: 'Address e-mail invalid'
```
Comment (szepeviktor) [inline sweetalert2.d.ts:1132]: I am not sure here. Is it French or English?

**Comment by szepeviktor** [comment]
This dump AI rabbit speaks misleading things!

**Comment by szepeviktor** [comment]
@coderabbitai ignore

**Comment by szepeviktor** [comment]
@coderabbitai remove all your comments

**Review by limonte — APPROVED** [review]
Thank you @szepeviktor 🙌

**Comment by szepeviktor** [comment]
Glad to contribute.

**Pull request merged**